tests/operator_test.go

```diff
@@ -212,6 +232,39 @@ var _ = Describe("Operator delete CDI tests", func() {
 		}, 2*time.Minute, 1*time.Second).Should(BeTrue())
 	})
 
+	It("[test_id:3954]should delete an upload pod", func() {
```

Comment:
How is this different from the test right above it on line 202? Maybe an accidental copy?

Reply:
Yes, you're right, removed the second copy.